### DISCUSSION TITLE
pushing room temp instead of pulling it?

### DISCUSSION THREAD

**Discussion by zachari043** [SwiCago/HeatPump#208] [General]
Hello! great work ! Just a little question, I know Mitsu unit can work with wired remote, that have it's own temp sensor, so it may be possible to send external room temp to te unit bypassing it's own sensor, have you investigate how to disable internal sensor and sending temp from external source ?

**Comment by SwiCago** [comment]
The library has support for external temp sensors. 
Use method setRemoteTemperature(float celsius)
Send setRemoteTemperature(0) to revert back to internal sensors.
